jsk_pcl_ros/CMakeLists.txt

```diff
@@ -626,6 +628,7 @@ if (CATKIN_ENABLE_TESTING)
     add_rostest(test/test_voxel_grid_downsample.test)
     add_rostest(test/test_voxel_grid_large_scale.test)
     add_rostest(test/test_organized_statistical_outlier_removal.test)
+    add_rostest(test/test_container_occpancy_detector.test)
```

Comment:
Test fails because of file name typo.
`occpancy` ==> `occupancy`

Reply:
sorry, fixed